### ISSUE TITLE
SAR: attract to sugar

### ISSUE THREAD

**Issue by ericsoco** [stamen/living-liquid#41]
Implement for real. Right now it just swims around in a circle.

**Comment by nicolettehayes** [comment]
Feedback from Elisha:

Also, movement should ALWAYS be directed at the nearest sugar source, if that is possible (chemotaxis works on a distance/concentration gradient). I would have it move around through early evening until the sugar is cleared, then stop - drifting only - and divide.

Precise timing for movement to sugar in spreadsheet: https://docs.google.com/a/stamen.com/spreadsheets/d/1N0DhkQ9osKbXjEREfkXH4eb6_dQ-3Kn76n8Wcj9Cc9o/edit?usp=sharing

![screen shot 2017-03-07 at 5 09 40 pm](https://cloud.githubusercontent.com/assets/12550451/23685461/01860470-0359-11e7-940a-df12d665db10.png)

**Comment by joycema** [comment]
Yes... so, they do not randomly swim around but always towards sugar.  (Don't think that is currently modeled.)

**Comment by joycema** [comment]
To Elisha's 2nd point --- SAR116 continues to swim until all the sugar disappears.   So, all the sugar needs to disappear to correspond with the time the SAR116 stops swimming in the sandbox and AND the timing indicated in its control panel's bar graph.  I tried to get this to happen by changing values in particle.json.  But, please look this over to see if it's convincing.

Also, SAR116 does not start swimming until sugar is released --- I changed this in the particle.json file to match the start swimming and eating with the start releasing --- there was an error mismatch earlier.  So, this part is correct now.
